src/app/src/main/kotlin/org/imperial/mrc/comet/controllers/DataController.kt

```diff
@@ -13,6 +14,12 @@ class DataController(
     private val logger: Logger = LoggerFactory.getLogger(DataController::class.java),
     private val apiClient: APIClient
 ) {
+    @GetMapping("/countries")
+    fun countries(): ResponseEntity<String> {
+        logger.info("countries")
```

Comment:
I vaguely remember playing with a filter to automatically log requests + params. Will dig it out in a maintenance ticket.